### DISCUSSION TITLE
Auth Options

### DISCUSSION THREAD

**Discussion by kagulati09** [requarks/wiki#4719] [Help / Questions]
Hi, I have the authentication on with Discord. I was wondering if there is a way to disable the login option via The Local Database.
I was also wondering if there is a way to switch the authentication provider for a user. I am the owner of the wiki and I am unable to switch from The Local Database to Discord

**Comment by whiteatom** [comment]
I just hide the local auth under Admin > System/Security > Login > Hide Local Auth and then enabled bypass login screen and my users are taken right to the external auth screen. 

I couldn't figure out how to change the authorization provider  for my account. I had to take my new account through my external auth and add it to the admin group through SQL.
